workflows/publishVSCode.yml

```
@@ -0,0 +1,47 @@
+name: Publish VSCode Extension
+
+on:
+  release:
+    types: [created]
```

```suggestion
  release:
    types: [released]
  pull_request:
    branches: [ '**' ]
```
the `created` event type seems a little loose, whereas `released` will only trigger when the release is for real, not draft or prerelease.

I'm suggesting the `pull_request` event too in an attempt to create a dry-run/validation for creation of the VSIX file, and we would just add to the conditional in the `Publish Extension` step to skip is the job is run in a PR.